### PR TITLE
Add fast-jar support for Quarkus Maven plugin

### PR DESCRIPTION
**Pull request by zanmagerl**
This resolves the issue #92. 
Similar was done in #129 for Gradle plugin, but this is for the Maven plugin. Most of the code is therefore more or less the same.

Sidenote: when will be a new release of the aforementioned Gradle Quarkus plugin?

### CODE REVIEW

**Comment by elefeint** [comment]
The lack of release was my oversight, sorry. It's [released now](https://repo1.maven.org/maven2/com/google/cloud/tools/jib-quarkus-extension-gradle/0.1.2/).

**Comment by mpeddada1** [comment]
Thank you for your contribution! This is looking good. I've added some minor comments. Please let me know if you have any questions.

File: first-party/jib-quarkus-extension-maven/src/test/java/com/google/cloud/tools/jib/maven/extension/quarkus/JibQuarkusExtensionTest.java [hunk 1]
```diff
@@ -165,7 +167,45 @@ public void testExtendContainerBuildPlan_noQuarkusRunnerJar() throws IOException
...
+  public void testExtendContainerBuildPlan_noQuarkusFastJar() throws IOException {
+    ContainerBuildPlan buildPlan = ContainerBuildPlan.builder().build();
+    properties.put("packageType", "fast-jar");
+
+    try {
+      new JibQuarkusExtension()
+          .extendContainerBuildPlan(buildPlan, properties, Optional.empty(), mavenData, logger);
+      fail();
+    } catch (JibPluginExtensionException ex) {
+      assertEquals(JibQuarkusExtension.class, ex.getExtensionClass());
```
[the remark this answers is not in the document]
Comment (zanmagerl) [reply first-party/jib-quarkus-extension-maven/src/test/java/com/google/cloud/tools/jib/maven/extension/quarkus/JibQuarkusExtensionTest.java:196]: Good suggestion, I will also fix it for other tests in this file.

**Comment by zanmagerl** [comment]
> Thank you for your contribution! This is looking good. I've added some minor comments. Please let me know if you have any questions.

Thanks for reviewing and for suggestions for improvements! Fixes are present in another commit.

**Review by mpeddada1 — APPROVED** [review]
LGTM! Thanks for addressing all the comments!

**Pull request merged**